_episodes/03-create.md

```diff
@@ -143,6 +143,16 @@ wording of the output might be slightly different.
 >
 > > ## Solution -- USE WITH CAUTION!
 > >
+> > ### Background
+> > Removing files from a git repository needs to be done with caution. To remove files from the working tree and not from your working directory, use
```

Comment:
In addition to #588, I think this line is very confusing. Do you mean the staging area / index, where you write "working tree"?